test/fixtures/simple_regional_with_networking/example.tf

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "random_string" "suffix" {
+  length  = 4
+  special = false
+  upper   = false
+}
+locals {
+  network           = "gke-network-${random_string.suffix.result}"
```

Comment:
No reason to store these in locals, you could simply pass them through directly.

However, I actually suggest we *don't* explicitly set these but instead use the `defaults` I suggested adding above.

Reply:
Done using defaults as suggested